deploy/csi-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
```

Comment:
Please add  deployment yaml files by creating a template file in `chart/templates/` and running `script/generate-deploy-yamls.sh`

Reply:
@blaisedias We will leave that part for later, we are expecting a lot of churn in these files still.

Reply:
If we do, it makes it difficult to automate running the tests with control plane 2.